charts/external-es-proxy/templates/_helpers.tpl

```diff
@@ -101,3 +101,15 @@ imagePullSecrets:
 {{ .Values.images.awsproxy.repository }}:{{ .Values.images.awsproxy.tag }}
 {{- end }}
 {{- end }}
+
+
+
```

Comment:
`external-es-proxy-nginx-common` is an ambiguous name. Maybe we should rename it to `external-es-proxy-nginx-location-common` ? 

Also the templating I was talking about earlier was to reduce duplication between charts/elasticsearch/templates/nginx/nginx-es-configmap.yaml and charts/external-es-proxy/templates/external-es-proxy-configmap.yaml